packages/x-components/src/x-modules/url/components/url-handler.vue

```diff
@@ -165,6 +175,18 @@
       this.urlLoaded = true;
     }
 
+    /**
+     * Prevents the user from either typing or pasting special characters in the url query.
+     *
+     * @internal
+     * @param query - Query from the url that will be checked for special characters.
+     */
+    protected preventSpecialKey(query: string): void {
+      if (/[<>]/.test(query ?? '')) {
+        this.hasSpecialKeys = true;
```

Comment:
Why storing this value in `hasSpecialKeys` protected property instead of returning true/false in this function and then test it on line 167. I mean, I would change this function name to `hasSpecialKeys` and then test:

`if (!this.hasSpecialKeys(this query)) {`